### ISSUE TITLE
After the plugin update the software becomes very stuck and I can't switch between different notes

### ISSUE THREAD

**Issue by bbcghh123** [vrtmrz/obsidian-livesync#63]
After the plugin update the software becomes very stuck and I can't switch between different notes

**Comment by andreylitvintsev** [comment]
Plugin version [0.8.3](https://github.com/vrtmrz/obsidian-livesync/releases/tag/0.8.3). Same problem! Plugin not send anything into couchdb. Its reproduce on Android. PC - all great! 

Rollback to version 7.2.0 helps for me. 

P.S. Sorry for my english.

**Comment by vrtmrz** [comment]
Oh! I'm very sorry! 
In my environment, this is working now.

May I ask for your device information?
It would be helpful if you could copy-paste from `Show debug info`
e.g.
```
SYSTEM INFO:
	Obsidian version: v0.14.6
	Installer version: v0.12.15
	Operating system: Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_ARM64_T8101 21.3.0
	Login status: logged in
	Catalyst license: supporter
	Insider build toggle: off
	Live preview: on
	Legacy editor: on
	Base theme: light
	Community theme: Deep Work
	Snippets enabled: 2
	Safe mode: off
```

And, Enabling "Use old connection method" doesn't work?

**Comment by andreylitvintsev** [comment]
Sorry for long answer!
I can't toggle "Use old connection method" even after disabling "LiveSync" option. Obsidian application self become not responsive.

I can't find "Show debug info", where is placed?

**Comment by vrtmrz** [comment]
It sounds so serious.

`Show debug info` can be launched via command pallet (`Ctrl` + `P` or `Command` + `P`).
Did anything show on DevTools? (You can show DevTools by `Ctrl`+`Shift`+`I`)

**Comment by andreylitvintsev** [comment]
My Android. Can't open DevTools console on mobile.
```
SYSTEM INFO:
 Operating system: android 11 (Google Pixel 4a)
 Login status: not logged in
 Live preview: off
 Legacy editor: on
 Base theme: dark
 Community theme: none
 Snippets enabled: 0
 Safe mode: off
 Plugins installed: 1
 Plugins enabled: 1
  1: Self-hosted LiveSync
```

My PC (no problem on pc).  I check DevTools console - no errors or warnings.
```
SYSTEM INFO:
	Obsidian version: v0.14.6
	Installer version: v0.13.31
	Operating system: Windows 10 Pro 10.0.19044
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: on
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Safe mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Self-hosted LiveSync
```

**Comment by vrtmrz** [comment]
Thanks for your information!
On Android 12 Pixel 3a XL works fine. I think it is almost the same environment, it may be a configuration problem.

May I see your data.json under your-vault/.obsidian/plugins/obsidian-livesync/?
(Please delete the values of `couchDB_URI`,`couchDB_USER`,`couchDB_PASSWORD`,`couchDB_DBNAME`,`passphrase` and `workingPassphrase`)

If you are hard to find this file, please use [screwdriver](https://github.com/vrtmrz/obsidian-screwdriver).
The file can be dumped by this template:
```
---
# --- Select a directory to dump. ---
target: .obsidian/plugins/obsidian-livesync

# --- Regular expressions for filtering files
filters:
 - data.json
---
```

**Comment by andreylitvintsev** [comment]
```
{
  "couchDB_URI": "~~~",
  "couchDB_USER": "~~~",
  "couchDB_PASSWORD": "~~~",
  "couchDB_DBNAME": "~~~",
  "liveSync": true,
  "syncOnSave": false,
  "syncOnStart": false,
  "savingDelay": 200,
  "lessInformationInLog": false,
  "gcDelay": 300,
  "versionUpFlash": "",
  "minimumChunkSize": 20,
  "longLineThreshold": 250,
  "showVerboseLog": false,
  "suspendFileWatching": false,
  "trashInsteadDelete": true,
  "periodicReplication": false,
  "periodicReplicationInterval": 60,
  "syncOnFileOpen": false,
  "encrypt": false,
  "passphrase": "~~~",
  "workingEncrypt": false,
  "workingPassphrase": "~~~",
  "doNotDeleteFolder": false,
  "resolveConflictsByNewerFile": false,
  "batchSave": false,
  "deviceAndVaultName": "",
  "usePluginSettings": false,
  "showOwnPlugins": false,
  "showStatusOnEditor": false,
  "usePluginSync": false,
  "autoSweepPlugins": false,
  "autoSweepPluginsPeriodic": false,
  "notifyPluginOrSettingUpdated": false,
  "checkIntegrityOnSave": false,
  "batch_size": 250,
  "batches_limit": 40,
  "useHistory": false,
  "disableRequestURI": false
}
```

Thanks for you job!

**Comment by vrtmrz** [comment]
Interim report!

LiveSync(Synchronization method) causes freezing.
(I didn't notice this while I don't use LiveSync on mobile, really sorry...)
I'm finding why this has been caused. Until fixed up, please use another synchronization method (Just like periodic sync).

**Comment by mhdali** [comment]
Confirming that the periodic sync solves the problem for now. Thanks for all your efforts into this plugin!

**Comment by vrtmrz** [comment]
Thank you! I'm very relieved to hear that.
Currently, replication handling implementation is so complicated. I will rewrite this.
But I'll be a bit late to fix this problem completely, please hold on, sorry...

When mobile to desktop, Sync on save is almost the same as LiveSync. But it will not receive changes on the desktop in real-time.

**Comment by vrtmrz** [comment]
Finally, fixed on v0.8.6! 
It caused by the new Obsidian API on mobile.
I thought it doesn't matter to periodic sync, but in was affected all replication.
In mobile, “Use the old connecting method” is locked to enabled now.
Could you please try v0.8.6?

**Comment by vrtmrz** [comment]
Close this issue once. Feel free to open it again if you have new something.
